src/Object.zig

```diff
@@ -198,46 +198,46 @@ pub fn parse(self: *Object) !void {
             var i: u32 = 0;
             var offset: u32 = 0;
             while (i < nsyms) : (i += 1) {
-                const verdefsym: elf.Elf64_Verdef = if (self.is32Bit()) blk: {
-                    const verdefsym = @as(*align(1) const elf.Elf32_Verdef, @ptrCast(raw.ptr + offset)).*;
+                const verdefsym: elf.Verdef = if (self.is32Bit()) blk: {
+                    const verdefsym = @as(*align(1) const elf.Verdef, @ptrCast(raw.ptr + offset)).*;
                     break :blk .{
-                        .vd_version = verdefsym.vd_version,
-                        .vd_flags = verdefsym.vd_flags,
-                        .vd_ndx = verdefsym.vd_ndx,
-                        .vd_cnt = verdefsym.vd_cnt,
-                        .vd_hash = verdefsym.vd_hash,
-                        .vd_aux = verdefsym.vd_aux,
-                        .vd_next = verdefsym.vd_next,
+                        .version = verdefsym.version,
+                        .flags = verdefsym.flags,
+                        .ndx = verdefsym.ndx,
+                        .cnt = verdefsym.cnt,
+                        .hash = verdefsym.hash,
+                        .aux = verdefsym.aux,
+                        .next = verdefsym.next,
                     };
```

Comment:
I don't think these are needed anymore since both 32-bit and 64-bit ELF files expose the same structure.